### PR TITLE
Reformat value_is_nan

### LINKED ISSUE #4600: value_is_nan

### PR DESCRIPTION
**Pull request by usamafaisal98**
- [ ] #4600

closes #4600

### CODE REVIEW

**Comment by usamafaisal98** [comment]
@juliagsy could you review please?

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! The changes so far looks good! There's just a bit more do check, could you also add some examples to the equivalent instance methods in `array/general.py` and `container/general.py`? Please request my review again once you're done, thank you!

**Comment by usamafaisal98** [comment]
Hi @juliagsy Could you review again please? Thanks!

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -1317,9 +1317,62 @@ def value_is_nan(
     ret
         Boolean as to whether the input value is a nan or not.
 
+    Examples
+    --------
+    >>> x = ivy.array([451])
+    >>> y = x.value_is_nan()
```
Comment (juliagsy) [inline ivy/functional/ivy/general.py:1323]: sorry, I just realised, this should be `ivy.value_is_nan(x)` to showcase functional method

File: ivy/functional/ivy/general.py [hunk 2]
```diff
@@ -1317,9 +1317,62 @@ def value_is_nan(
...
+    With :code:`ivy.Container` input:
+    >>> x = ivy.Container(a=ivy.array([420.]), b=ivy.array([72]))
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    {
+        a: False,
+        b: False
+    }
+
+    With one :code:`ivy.Container` static method:
```
Comment (juliagsy) [inline ivy/functional/ivy/general.py:1356]: the examples starting from here are not needed as they are a repeat of the ones in `container/general.py`

File: ivy/container/general.py [hunk 3]
```diff
@@ -2529,6 +2558,31 @@ def value_is_nan(
         ret
             Boolean as to whether the input value is a nan or not.
 
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([425]), b=ivy.array([float('nan')]))
+        >>> y = x.value_is_nan()
+        >>> print(y)
+        {
+            a: False,
```
Comment (juliagsy) [inline ivy/container/general.py:2567]: somehow the generated strings for bool representation is lowercase (i.e., `false`, `true`), which currently causes doctoring test failures, would you mind changing the container representation returns to be lowercase? only the examples here, thanks a lot!

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey! Thanks for the changes! Just some small fix and test resolving left for a merge! As usual, please request my review once again when you're done! Thanks!

**Comment by usamafaisal98** [comment]
@juliagsy could you review again please? thanks!

**Comment by juliagsy** [comment]
Thanks for contributing!

**Pull request merged**